packages/assets-controllers/src/Standards/ERC20Standard.ts

```diff
@@ -119,14 +119,11 @@ export class ERC20Standard {
     decimals: string | undefined;
     balance: BN | undefined;
   }> {
-    const [decimals, symbol] = await Promise.all([
+    const [decimals, symbol, balance] = await Promise.all([
```

Comment:
also here? `Promise.allSettled`

Reply:
Same comment that I think we want to maintain the behavior of throwing on error, which the extra arg to `Promise.all` does.